uptane-standard.md

```diff
@@ -822,7 +836,7 @@ In order to perform full verification, an ECU SHALL perform the following steps:
 11. Verify that Targets metadata from the Director and Image repositories match. A primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the target metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
     1. Locate and download a Targets metadata file from the Image repository that contains an image with exactly the same file name listed in the Director metadata, following the procedure in {{resolve_delegations}}.
     2. Check that the Targets metadata from the Image repository matches the Targets metadata from the Director repository:
-        1. Check that the non-custom metadata (i.e., length and hashes) of the unencrypted image are the same in both sets of metadata.
+        1. Check that the non-custom metadata (i.e., length and hashes) of the unencrypted or encrypted image are the same in both sets of metadata.
```

Comment:
The image repository does not know anything about encrypted images, so it cannot have metadata about them.

Reply:
I think it is unrealistic to force an implementer to have unencrypted images on the repository.

I think this is an issue we should gain alignment on.

Reply:
I looked a little closer a this section/change.
This is included in the process for a Primary to verify metadata.
It states that the Primary shall check the Targest metadata from Image repo matches Director Repo [838].
This is further expanded upon to check the non-custom metadata, including length & hash on unencrypted image.
In the case that the image is encrypted with a key that **only** a specific secondary can decrypt, the primary will fail at this step.
Hence, my clarification above.

I will add: "the primary will validate encrypted images & metadata, but target ECU must validate the unencrypted image and its associated metadata."